docs/book/src/tasks/kubeadm-control-plane.md

```diff
@@ -20,10 +20,6 @@ with a valid lifespan of a year, and will be automatically regenerated when the
 
 See the section on [upgrading clusters][upgrades].
 
-#### Using Kubeadm Control Plane when upgrading from Cluster API v1alpha2 (0.2.x)
```

Comment:
I assume it's good enough that we cover this in the two older books, but if there are concerns it's absolutely fine for me to add it again.